scripts/cibuild

```diff
@@ -16,7 +16,12 @@ Build application for staging or a release.
 if [[ -n "${GIT_COMMIT}" ]]; then
     GIT_COMMIT="${GIT_COMMIT:0:7}"
 else
-    GIT_COMMIT="$(git rev-parse --short HEAD)"
+    # TODO A run of cipublish on Jenkins resulted in an image being published
+    # with an 8-character tag. Is this workaround the best solution? This SO
+    # answer indicates that --short does not guarantee a length of 7
```

Comment:
good find. This is implemented in a lot of places, so a bigger discussion is necessary about how to deal with this